tests/drawer/test_drawable_layers.py

```diff
@@ -140,22 +190,27 @@ def test_all_wires_measurement(self, measurement):
 
     def test_mid_measure_custom_wires(self):
         """Test that custom wires do not break the drawing of mid-circuit measurements."""
+        mp0 = MidMeasureMP("A", id="foo")
+        mp1 = MidMeasureMP("a", id="bar")
+        m0 = qml.measurements.MeasurementValue([mp0], lambda v: v)
+        m1 = qml.measurements.MeasurementValue([mp1], lambda v: v)
 
         def teleport(state):
             qml.QubitStateVector(state, wires=["A"])
             qml.Hadamard(wires="a")
             qml.CNOT(wires=["a", "B"])
             qml.CNOT(wires=["A", "a"])
             qml.Hadamard(wires="A")
-            m0 = qml.measure("A")
-            m1 = qml.measure("a")
+            qml.apply(mp0)
+            qml.apply(mp1)
```

Comment:
why was this change necessary?

Reply:
So that the `MidMeasureMP`s that I put in the bit map have the same ID as the original `MidMeasureMP`s.